build.sbt

```diff
@@ -486,9 +486,8 @@ lazy val commonSettings = instanceSettings ++ clearSourceAndResourceDirectories
   Compile / unmanagedResourceDirectories += (ThisBuild / baseDirectory).value / "src" / thisProject.value.id,
   sourcesInBase := false,
   Compile / scalaSource := (Compile / sourceDirectory).value,
-  Compile / javaSource := (Compile / sourceDirectory).value,
-  // resources are stored along source files in our current layout
-  Compile / resourceDirectory := (Compile / sourceDirectory).value,
```

Comment:
I'm more concerned about the silent impact of removing these for linting reasons.

Are we sure that CI (or yourself, manually) is good enough that dropping these two isn't dropping anything on the floor?  I'd rather opt the linter out, unless you're fairly confident.

Reply:
pleased to meet you, Cautious Dale! I'm better acquainted with your brother, Let’s-Just-Merge-It-and-See Dale, but I'm sure the three of us will get along just fine :-)

I admit to not being too worried about this change in particular, but actually you're right — this is a substantial version bump (1.3 -> 1.4, I mean), so it would be prudent to use `jardiff` on all the build products and see if any funny stuff turns up

and so I did that now, and nothing turned up

Reply:
Okie dokie, thank you!